verticaltabs.js

```diff
@@ -430,7 +430,7 @@ VerticalTabs.prototype = {
 
     let enter = (event) => {
       this.mouseEntered();
-      if (event.type === 'mouseenter' && leftbox.getAttribute('expanded') !== 'true') {
+      if ((event.type === 'mouseenter' || event.type === 'mouseover') && leftbox.getAttribute('expanded') !== 'true') {
         this.stats.tab_center_expanded++;
```

Comment:
We definitely don't want to add to the stats on `mouseover`.

Also, since we only call the `enter` function on `mouseenter` or `mouseover`, this forst part of the if condition should always be true.  Perhaps it would make more sense to pull the `setAttribute` call out of this block?